futures-util/Cargo.toml

```diff
@@ -12,7 +12,7 @@ Common utilities and extension traits for the futures-rs library.
 
 [features]
 std = ["futures-core/std", "futures-io/std", "futures-sink/std", "either/use_std"]
-default = ["std"]
+default = ["std", "futures-core/either"]
```

Comment:
Is this sufficient, or does the `futures-core` feature need to be handled in other ways?

Reply:
Seems fine to me!